prow/plugins/trigger/pull-request_test.go

```diff
@@ -289,95 +317,98 @@ func TestHandlePullRequest(t *testing.T) {
 	}
 	for _, tc := range testcases {
 		t.Logf("running scenario %q", tc.name)
-
-		g := &fakegithub.FakeClient{
-			IssueComments: map[int][]github.IssueComment{},
-			OrgMembers:    map[string][]string{"org": {"t"}},
-			PullRequests: map[int]*github.PullRequest{
-				0: {
-					Number: 0,
-					User:   github.User{Login: tc.Author},
-					Base: github.PullRequestBranch{
-						Ref: "master",
-						Repo: github.Repo{
-							Owner: github.User{Login: "org"},
-							Name:  "repo",
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
I added this as it was hard to see which test failed.
Would be ok with reverting this again.

Reply:
Please do keep it :)